src/main/proto/halo_cmm/uk/pilot/common_template.proto

```diff
@@ -27,4 +27,27 @@ message Common {
     display_name: "Common Template"
     description: "A common base template for Origin Event filtering."
   };
+
+  enum Sex {
```

Comment:
Do we need add OTHER? (in addition to UNSPECIFIED)

Reply:
for Meta, we're ok with just the 3 as we only have male/female/unknown outputs from our labeler, and it matches what we have here https://github.com/world-federation-of-advertisers/virtual-people-common/blob/main/src/main/proto/wfa/virtual_people/common/demographic.proto#L23, unless there are plans to update that too soon.

but i know that might be different for other EDPs

Reply:
I'm checking the requirements based on the output expected. @emadlivani  does google support Other/unkwnon as a category?

Reply:
We can leave it as is.